setup.py

```diff
@@ -1,4 +1,6 @@
-# Copyright 2018 Google LLC
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Google LLC
```

Comment:
The copyright year shouldn't be updated.

Reply:
Fixed